src/main/java/com/microfocus/octane/gitlab/helpers/CIBuildStatusInfoImpl.java

```diff
@@ -0,0 +1,120 @@
+package com.microfocus.octane.gitlab.helpers;
+
+import com.hp.octane.integrations.dto.general.CIBuildStatusInfo;
+import com.hp.octane.integrations.dto.parameters.CIParameter;
+import com.hp.octane.integrations.dto.snapshots.CIBuildResult;
+import com.hp.octane.integrations.dto.snapshots.CIBuildStatus;
+
+import java.util.List;
+
+public class CIBuildStatusInfoImpl implements CIBuildStatusInfo {
```

Comment:
please use this one instead of your class:
com.hp.octane.integrations.dto.general.impl.CIBuildStatusInfoImpl